plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -53,4 +53,7 @@ private MLCommonsSettings() {}
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         );
+
+    public static final Setting<Integer> ML_COMMONS_NATIVE_MEM_THRESHOLD = Setting
+        .intSetting("plugins.ml_commons.native_mem_threshold", 90, 0, 100, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
`mem` seems not so clear. how about change to 
```suggestion
        .intSetting("plugins.ml_commons.native_memory_threshold", 90, 0, 100, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Reply:
sure, will change it to memory.